src/DefaultBuilder/DefaultBuilder.sln

```diff
@@ -0,0 +1,173 @@
+﻿
```

Comment:
My hunch is that you'll need to add a lot more projects to this solution to workaround NU1105.